src/wrapperspkg/formulas/soloud/wrapper.nim

```diff
@@ -11,20 +11,12 @@ const
 static:
   gitPull("https://github.com/jarikomppa/soloud", baseDir, "include/*\nsrc/*\n")
 
-cDisableCaching()
```

Comment:
You lost this.

Reply:
was intentional; isn't that only required for testing inside nimterop? 
ie assuming https://github.com/nimterop/nimterop/issues/69 gets fixed, there should be no need for that (except for testing that caching feature, which should be properly tested by nimterop, not `wrappers`); 

until https://github.com/nimterop/nimterop/issues/69 is fixed, at very least client code shouldn't have that enabled in the normal course IMO; so maybe i can have:
`
when defined(nimterop_forcerebuild) cDisableCaching()
` 
thoughts?

Reply:
=> tracked in https://github.com/nimterop/nimterop/issues/108